elastic/kibana
# [Fleet] Fix Install with streaming to save readme and icons

## Summary

Fix installing security_detection_engine with streaming to save related readme, changelog and icons in ES, as once the package is installed we load the assets from ES and not the registry.

## How to test?

Install `security_detection_engine` (reload without cache) and verify image and reamde are correctly accessible in the integrations app for `security_detection_engine` 
<img width="1488" alt="Screenshot 2024-11-11 at 8 49 16 PM" src="https://github.com/user-attachments/assets/bc10f008-d9ef-4c11-8ba0-b17f4ca389b0">
